VocableUITests/Tests/PresetPhraseTests.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  CustomCategoriesTests.swift
+//  VocableUITests
+//
+//  Created by Canan Arikan and Rudy Salas on 04/19/20.
+//  Copyright © 2022 WillowTree. All rights reserved.
+//
+import XCTest
+
+class PresetPhraseTests: BaseTest {
+    
+    func testAddNewPhrase() {
+        let customPhrase = "Add"
+        let category = "Environment"
+                
+        // Navigate to our test category and Add a phrase
+        SettingsScreen.navigateToSettingsCategoryScreen()
+        SettingsScreen.openCategorySettings(category: category)
+        CustomCategoriesScreen.editCategoryPhrasesButton.tap()
+        CustomCategoriesScreen.addPhrase(customPhrase)
+        
+        // Verify that phrase does exist in Category Details Screen
+        XCTAssertTrue(CustomCategoriesScreen.doesPhraseExist(customPhrase))
+        
+        // Verify that phrase doesn exist in Main Screen
+        CustomCategoriesScreen.returnToMainScreenFromEditPhrases()
+        MainScreen.locateAndSelectDestinationCategory(.environment)
+        XCTAssertTrue(MainScreen.doesPhraseExist(customPhrase))
+    }
+    
+    func testEditPresetPhrase() {
+        let customPhrase = "ab"
+        let category = "Personal Care"
+                
+        // Navigate to our test category
+        SettingsScreen.navigateToSettingsCategoryScreen()
+        SettingsScreen.openCategorySettings(category: category)
+        CustomCategoriesScreen.editCategoryPhrasesButton.tap()
+        
+        // Define the query that gives us the first phrase listed
+        let originalPhraseId = XCUIApplication().cells.firstMatch.identifier
+        let originalPhraseCell = XCUIApplication().cells[originalPhraseId]
+        let originalPhrase = originalPhraseCell.staticTexts.firstMatch.label
+        
+        //firstPhraseCell.tap()
+        originalPhraseCell.staticTexts[originalPhrase].tap()
+        KeyboardScreen.typeText(customPhrase)
+        KeyboardScreen.checkmarkAddButton.tap()
+   
+        let updatedPhrase = originalPhrase + customPhrase
+        // Verify that preset phrase doesn't exist in Category Details Screen
+        XCTAssertFalse(CustomCategoriesScreen.doesPhraseExist(originalPhrase))
+        
+        // Verify that edited phrase exists in Category Details Screen
+        XCTAssertTrue(CustomCategoriesScreen.doesPhraseExist(updatedPhrase))
+        
+        // Verify that preset phrase doesn't exist in Main Screen
+        CustomCategoriesScreen.returnToMainScreenFromEditPhrases()
+        MainScreen.locateAndSelectDestinationCategory(.personalCare)
+        
+        // Verify that edited phrase exists in Main Screen
+        XCTAssertTrue(MainScreen.doesPhraseExist(updatedPhrase))
```

Comment:
Since we are no longer checking that the original phrase appears on the main screen, we can remove the extra comment and consolidate the code block, like this:
```suggestion
        // Verify that updated phrase exists in Main Screen
        CustomCategoriesScreen.returnToMainScreenFromEditPhrases()
        MainScreen.locateAndSelectDestinationCategory(.personalCare)
        XCTAssertTrue(MainScreen.doesPhraseExist(updatedPhrase))
```